File: C/2709.c
#include <stdio.h>

int main(){

    int m, passoSomatoria;

    while(scanf("%i", &m) != EOF){

        int valoresMoedas[m], soma = 0, numDivisores = 0;

        for(int i = 0; i < m; i++){
            scanf("%i", &valoresMoedas[i]);
        }

        scanf("%i", &passoSomatoria);

        for (int i = m-1; i >= 0; i -= passoSomatoria){
            soma += valoresMoedas[i];
        }

        for(int i = 1, b = soma+1; i < b; i++ ){
            if(soma % i == 0){
                numDivisores += 1;
            } 
        }

        if(numDivisores == 2){
            printf("You’re a coastal aircraft, Robbie, a large silver aircraft.\n");
        }else{
            printf("Bad boy! I’ll hit you.\n");
        }

    }

    return 0;
}

File: Python/2709.py
while True:
    try:

        m = int(input())

        valoresMoedas = []

        for i in range(m):
            valoresMoedas.append(int(input()))

        passoSomatoria = int(input())

        soma = 0

        for i in range(m-1, -1, -passoSomatoria):
            soma += valoresMoedas[i]

        numDivisores = 0

        for i in range(1, soma+1):
            if soma % i == 0:
                numDivisores += 1

        if numDivisores == 2:
            print("You’re a coastal aircraft, Robbie, a large silver aircraft.")
        else:
            print("Bad boy! I’ll hit you.")

    except:
        break